awe_languagetool/setup/data.py

```diff
@@ -47,8 +47,11 @@ def extra_build_commands(develop=False, install=False):
         # approach.                                                     #
         #################################################################
 
-        dir_name = \
-            resources.path('awe_languagetool', '')
+        # Since python 3.11, we cannot use 'path', but this:
+        with resources.as_file(resources.files('awe_languagetool')) as path:
```

Comment:
Does this still work with version pre 3.11? Some systems may still be running 3.9.

Reply:
No, we are essentially committing to 3.11 at this point since 3.12 is the standards and the few dependencies for pre 3.10 are even being replaced.

Reply:
I asked this question because according to the [documentation](https://docs.python.org/3/library/importlib.resources.html#importlib.resources.as_file), the `as_file` method was released in 3.9. Only in 3.11 did the `path` function become deprecated. This implies that it should still work on 3.9.

If there are other dependency issues with 3.9 here, what are they and where are they documented?

Reply:
As far as I know, there were no dependency issues other than investigating importlib.resources.path for python3.11.

I'm currently investigating dependency issues with spacy & spacytextblob after moving from python 3.9 to 3.11, but those changes are not included in this pull request.

@bradley-erickson Is it incorrect to assume that we'll be moving all of AWE to python 3.11? That's was my impression thus far.

Reply:
Ideally, the system works on multiple versions of Python. I believe that we came to an agreement that 3.11 was going to be the new base version to use and that's fine.

I think that not version-locking as much as we can is better since that may result in more dependency hell down the line. If this code still works on 3.9, why version lock to 3.11 just for the rest of the codebase? What if someone wants to use this module independently?

Reply:
@bradley-erickson I've checked that AWE_LanguageTool works on both 3.9 and 3.11 versions; hopefully that addresses the above comment for seeing that this should still work for 3.9!